storage/dropbox/storage_dropbox.go

```diff
@@ -85,61 +73,43 @@ func (backend *StorageDropbox) Description() string {
 
 // AvailableSpace returns the free space on this backend
 func (backend *StorageDropbox) AvailableSpace() (uint64, error) {
-	account, err := backend.db.GetAccountInfo()
+	space, err := backend.dropy.Client.Users.GetSpaceUsage()
 	if err != nil {
 		return 0, err
 	}
-
-	return uint64(account.QuotaInfo.Quota - account.QuotaInfo.Shared - account.QuotaInfo.Normal), nil
+	return space.Allocation.Allocated, nil
```

Comment:
This should probably be: `space.Allocation.Allocated - space.Allocation.Used`.

Reply:
Yes right :sweat_smile: made a silly mistake here.
Fixed in https://github.com/knoxite/knoxite/pull/61/commits/696b40c110126e5f065848f3ea1e5a9d3c0701d9